### PR TITLE
Import code

### PR DESCRIPTION
**Pull request by rupertford**
Ready for review ...

### CODE REVIEW

**Comment by arporter** [comment]
Legally-approved license added to source files.
Repository structured in a form suitable for pypi (although not yet configured for it).
I can get the tests to run (all pass or xfail) by adding fgenerator/src to my PYTHONPATH (and installing fparser using pip).

**Review by arporter — CHANGES_REQUESTED** [review]
All looks OK - see earlier comment on this request.
My only issue is with having a file named 'subclass.py'. Maybe it should be something more descriptive such as 'fparser_wrapper.py'

**Comment by rupertford** [comment]
I've changed the name of the file as suggested by Andy.

File: src/fgenerator/base.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
 from fparser.block_statements import Select
 from fparser.statements import Case
 
-from fgenerator.subclass import OMPDirective
+from fgenerator.fparser_wrapper import OMPDirective
```
Comment (arporter) [inline src/fgenerator/base.py]: I like this - it's clearer what OMPDirective does now.

**Review by arporter — APPROVED** [review]
All tests pass or xfail. File has been re-named as suggested.

**Pull request merged**